app/models/agents/http_status_agent.rb

```diff
@@ -60,11 +61,12 @@ def receive(incoming_events)
     private
 
     def check_this_url(url)
-      if result = ping(url)
-        create_event payload: { 'url' => url, 'status' => result.status.to_s, 'response_received' => true }
-        memory['last_status'] = result.status.to_s
+      measured_result = TimeTracker.track { ping(url) }
```

Comment:
Yes, it makes total sense and I was being extremely lazy for not covering with tests 😄